components/pageDescription/PageDescription.js

```diff
@@ -0,0 +1,22 @@
+import { getString } from '../../lib/richText';
+import {
+  PageDescriptionContainer,
+  DescriptionContainer,
+  Description,
+  Img,
+} from './PageDescription.styles';
+import LongSparkArrow from '../longSparkArrow/LongSparkArrow';
+
+export default function PageDescription({ description, arrowText, img }) {
+  return (
+    <div>
+      <PageDescriptionContainer>
+        <LongSparkArrow arrowText={arrowText} onClick={() => {}} />
+        <DescriptionContainer>
+          <Description>{getString(description)}</Description>
+        </DescriptionContainer>
+      </PageDescriptionContainer>
+      <Img src={img} />
```

Comment:
Can you add an alt here? It would come from the CMS I believe if it's a CMS image